app/services/doc_auth/errors.rb

```diff
@@ -121,7 +123,7 @@ module Errors
       SELFIE_FAILURE => { long_msg: SELFIE_FAILURE, field_msg: SELFIE_FAILURE, hints: false },
       SELFIE_NOT_LIVE => { long_msg: SELFIE_NOT_LIVE, field_msg: SELFIE_FAILURE, hints: false },
       SELFIE_POOR_QUALITY => { long_msg: SELFIE_POOR_QUALITY, field_msg: SELFIE_FAILURE, hints: false },
```

Comment:
do we need to freeze the hashes inside here? like a `.each { |_k, v| v.freeze }.freeze` kinda thing?

Reply:
may as well, done!